manifest.json

```
@@ -37,8 +40,9 @@
     }
   ],
   "background": {
-    "scripts": ["src/bg/background.js"],
-    "persistent": false
+    "service_worker": "service_worker.js"
   },
-  "content_security_policy": "script-src 'self' https://www.googleapis.com; object-src 'self'"
+  "content_security_policy": {
+    "extensions": "script-src 'self' https://www.googleapis.com; object-src 'self'"
```

A few of changes:

- the property should be `extension_pages` per https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#content-security-policy
- use the `default-src 'self'` catch-all directive
- use the CrUX-specific host for the `connect-src` directive

```suggestion
    "extension_pages": "default-src 'self'; connect-src https://chromeuxreport.googleapis.com;"
```